CHANGELOG.md

```diff
@@ -5,21 +5,40 @@ FEATURES:
   The default behavior for this feature is `enabled: true`.
   See [https://www.consul.io/docs/k8s/health](https://www.consul.io/docs/k8s/health) for more information.
   In order to enable this feature for existing installations it is required to restart all connect injected deployments so that they are re-injected.
-  Until this is done health checks for these deployments will not be synced to Consul.
+  Until this is done, health checks for these deployments will not be synced to Consul.
 
   **It is recommended to enable TLS with this setting enabled because it requires making calls to Consul clients across the cluster.
     Without TLS enabled, these calls could leak ACL tokens should the cluster network become compromised.**
+* Support for custom resource definitions (CRDs) is now generally available.
+  CRDs require Consul >= 1.8.4. If you wish to use `ServiceIntentions`
+  custom resources then this requires Consul >= 1.9.0 (which is still in beta as of this release).
+
+  To enable, set `controller.enabled: true` in your Helm configuration:
+
+  ```yaml
+  controller:
+    enabled: true
+  ```
+
+  See [https://www.consul.io/docs/k8s/crds](https://www.consul.io/docs/k8s/crds)
+  for more information.
 
 BREAKING CHANGES:
 * This helm release only supports consul-k8s versions 0.20+
-* With the addition of the connect-inject health checks controller any connect services which have failing Kubernetes readiness
+* With the addition of the connect-inject health checks controller, any connect services which have failing Kubernetes readiness
   probes will no longer be routable through connect until their Kubernetes health probes are passing.
   Previously, if any connect services were failing their Kubernetes readiness checks they were still routable through connect.
   Users should verify that their connect services are passing Kubernetes readiness probes prior to using health checks synchronization.
+* When health checks are enabled, Consul clients will have `check_update_interval` set to `0s`. Previously,
+  it was set to its default of `5m`. This change ensures the output of the check will show up in the Consul UI immediately. [[GH-674](https://github.com/hashicorp/consul-helm/pull/674)]
+* CRDs: controller default `limits.memory` increased from `30Mi` to `50Mi` and `requests.memory` increased from `20Mi` to `50Mi`
+  based on observed usage. [[GH-649](https://github.com/hashicorp/consul-helm/pull/649)]
```

Comment:
🤔 is this a breaking change if it's for a beta feature?

Reply:
Not sure, I don't mind either way but thought it best to err on the side of breaking change.

Reply:
👍 let's keep it then just in case.